src/common/TheNavbar.vue

```diff
@@ -34,19 +48,34 @@ import { defineComponent } from "vue";
 
 import TheLogo from "@/common/TheLogo.vue";
 import AppIcon from "@/common/AppIcon.vue";
+import AppButton from "@/common/AppButton.vue";
 import { routes } from "../router";
 
+import userClient from "../user/client";
+
 export default defineComponent({
   name: "TheNavbar",
-  components: { AppIcon, TheLogo },
+  components: { AppIcon, TheLogo, AppButton },
   data() {
     return {
       isNavOpen: false
+      // isLoggedIn: true
     };
   },
   computed: {
     links() {
       return routes;
+    },
+    isLoggedIn() {
+      return userClient.currentUser() != null;
+    }
+  },
+  methods: {
+    logout() {
+      userClient.logout();
+    },
+    login() {
```

Comment:
I'm still new to all of this but this might help?

https://v3.vuejs.org/api/options-lifecycle-hooks.html#beforecreate